.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   test-bot:
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
```

Comment:
curious, why?

Reply:
CI started failing and I cannot determine the cause yet, was testing to see if it was just ubuntu or both. Only ubuntu is failing, probably because we only install the linux version. What I need to do is detect which OS we are on and install the appropriate executable.

https://github.com/bastionzero/homebrew-tap/actions

Reply:
https://rubydoc.brew.sh/OS.html#linux%3F-class_method